float32 y_joint
float32 x_joint
float32 alpha_joint
float32 beta_joint
float32 d_joint
float32 x_ee
float32 y_ee
float32 z_ee

bool homing_completed
bool last_position_reached
